libbeat/cmd/instance/beat.go

```diff
@@ -96,11 +95,12 @@ type beatConfig struct {
 	Seccomp  *common.Config `config:"seccomp"`
 
 	// beat internal components configurations
-	HTTP          *common.Config `config:"http"`
-	Path          paths.Path     `config:"path"`
-	Logging       *common.Config `config:"logging"`
-	MetricLogging *common.Config `config:"logging.metrics"`
-	Keystore      *common.Config `config:"keystore"`
+	HTTP            *common.Config `config:"http"`
+	Path            paths.Path     `config:"path"`
+	Logging         *common.Config `config:"logging"`
+	MetricLogging   *common.Config `config:"logging.metrics"`
+	Keystore        *common.Config `config:"keystore"`
+	Instrumentation *common.Config `config:"instrumentation"`
```

Comment:
I know there are no clear rules, but I'd prefer to use concrete types and only use common.Config if we really need it (e.g. capture a configuration without interpreting it yet). You might consider to change this line to `Instrumentation instrumentation.Config`).